<?xml version="1.0"?>
<robot name="robot"
  xmlns:xacro="http://www.ros.org/wiki/xacro">

  <xacro:include filename="$(find robot_description)/urdf/macros/link_geometry.urdf.xacro" />
  <xacro:include filename="$(find robot_description)/urdf/macros/parts/base_robot.urdf.xacro" />
  <xacro:include filename="$(find robot_description)/urdf/macros/properties_robot.urdf.xacro" />

  <!-- link: base_footprint -->
  <link name='base_footprint'/>
  
  <!-- ##### -->
  <!-- Robot -->
  <!-- ##### -->

  <!-- Base -->
  <xacro:base name="base_link" parent="base_footprint">
      <origin xyz="0 0 0" rpy="0 0 0"/>
  </xacro:base>

</robot>
